lib/Timer/src/timer.cpp

```diff
@@ -0,0 +1,16 @@
+#include "timer.h"
+
+Timer::Timer() {}
```

Comment:
Implement the constructor by adding an argument (unsigned long nextTimeOut=0), and then assign the member variable *nextTimeOut* to this argument.  Two ways;
1) Timer::Timer(unsigned long nextTimeOut = 0) : _nextTimeOut(nextTimeOut) {} //This is called an initialization list, convenient for small constructors like this (just assigning one field)
2) Timer::Timer(unsigned long nextTimeOut = 0)  {
      _nextTimeOut = nextTimeOut;
}